Changes.md

```diff
@@ -19,6 +19,7 @@ Improvements
   - Improved layout of Box node plug creator visibility toggles.
 - ArnoldShader : Moved the `toon` shader's `*_tonemap_hue_saturation` parameters to appropriate sections in the UI.
 - File Browser : The "Type" column can now be sorted. This sorts directories separately from files, which are sorted by their extension.
+- NumericWidget : Added the ability to use the mouse wheel to adjust values in the same manner as <kbd>Up</kbd> and <kbd>Down</kbd> (#6009).
```

Comment:
This will need to move up to 1.4.x.x as 1.4.12.0 has been released.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/6045/commits/c51f7e045ba402ab805ff739c82c3b3577f53c10